docs-site/content/ja/db/migration-patterns.md

```
@@ -0,0 +1,236 @@
+ls -la $(path/to/migrations)
+```
+
+2. ファイルパターンを確認 — ファイル拡張子が期待されるパターンと一致するか
+
+3. 言語検出を確認:
+
+```bash
+cat .moai/config/sections/language.yaml
+```
+
+4. カスタムパスを設定:
```

markdownlint-cli2 (0.22.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 4; Style: 1/1/1

(MD029, ol-prefix)